src/Auction.hs

```diff
@@ -163,7 +163,7 @@ extractLastCall =
 
 -- displayLastCall is for use in explanations: it formats the most recent call
 -- from an action while stripping out any alerts it might have
-displayLastCall :: OutputType -> Action -> String
-displayLastCall fmt = output fmt . T.removeAlert . extractLastCall
+displayLastCall :: Action -> Commentary
+displayLastCall = toCommentary . T.removeAlert . extractLastCall
```

Comment:
I'd like to make `Action` a `Showable` type and rename `displayLastCall` to `toCommentary`, but the naive approach complains that it's an orphan instance because `Action` is just a synonym for `State Auction ()` and neither `State` nor `Showable` is defined in this file (even though `Action` is defined here, and that's the one I want to define things on). Maybe I need to make `Action` a new type on its own? but then it's unclear how many times I need to add the `Action` constructor elsewhere. This is complicated enough to be its own PR later.